ecs-cli/modules/cli/local/network/mock_network/setup.go

```diff
@@ -18,6 +18,7 @@
 package mock_network
 
 import (
+	io "io"
```

Comment:
I'm curious about the use of aliases here. Seems like we alias all imports regardless of whether there's a package name conflict. Is it because we want to make it possible to change the actual package used in the future without changing the setup code?

Reply:
I think this is because of something Efe has set up in his IDE

Aliasing "io" as io does seem kind of silly...

Reply:
lol this is generated with `mockgen` I didn't write this stuff :P